README.md

```diff
@@ -5,8 +5,6 @@ Tools, scripts and other systems infrastructure used to support the work of OME
 If you are interested in this work or require more information, please
 [contact us](http://www.openmicroscopy.org/site/community).
 
-[![Build Status](https://travis-ci.org/openmicroscopy/infrastructure.png)](http://travis-ci.org/openmicroscopy/infrastructure)
```

Comment:
We have status badges for GHA elsewhere, no? cc: @sbesson

Reply:
:+1: I am in favor for having CI/deployment badges whenever possible